tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -574,34 +542,32 @@ def test_positive_virt_card(
             {
                 'name': cr_name,
                 'provider': FOREMAN_PROVIDERS['vmware'],
-                'provider_content.vcenter': module_vmware_settings['vcenter'],
-                'provider_content.user': module_vmware_settings['user'],
-                'provider_content.password': module_vmware_settings['password'],
-                'provider_content.datacenter.value': module_vmware_settings['datacenter'],
+                'provider_content.vcenter': settings.vmware.vcenter,
+                'provider_content.user': settings.vmware.user,
+                'provider_content.password': settings.vmware.password,
+                'provider_content.datacenter.value': settings.vmware.datacenter,
                 'locations.resources.assigned': [module_location.name],
                 'organizations.resources.assigned': [module_org.name],
             }
         )
         session.hostgroup.update(hostgroup_name, {'host_group.deploy': cr_name + " (VMware)"})
         session.computeresource.vm_import(
             cr_name,
-            module_vmware_settings['vm_name'],
+            settings.vmware.vm_name,
```

Comment:
We are having it twice
```suggestion
            
```

Reply:
@shubhamsg199 its still duplicated, can you update this?

Reply:
Yes, updated now